ddtrace/settings/config.py

```diff
@@ -415,7 +424,32 @@ def __getattr__(self, name):
         if name not in self._config:
             self._config[name] = IntegrationConfig(self, name)
 
-        return self._config[name]
+        if name not in self._integration_configs:
+            self._integration_configs[name] = IntegrationConfig(self, name)
+
+        return self._integration_configs[name]
+
+    def _add_extra_service(self, service_name: str) -> None:
+        if self._extra_services_queue is None:
+            return
+        if self._remote_config_enabled and service_name != self.service:
+            try:
+                self._extra_services_queue.put_nowait(service_name)
+            except BaseException:  # nosec
+                pass
```

Comment:
Is there a volume reason for us not to log anything here? Catching exceptions and passing silently sounds like something someone will end up having to debug later on.

Reply:
@christophe-papazian  who wrote the original fix?